### PR TITLE
Adding livox_ros_driver to documentation index for distro

### PR DESCRIPTION
**Pull request by Livox-SDK**
Adding livox_ros_driver to documentation index for distro

### CODE REVIEW

File: kinetic/distribution.yaml [hunk 1]
```diff
@@ -5980,6 +5980,25 @@ repositories:
...
+      url: https://github.com/Livox-SDK/Livox-SDK-ROS.git
+      version: master
+    release:
+      packages:
+      - display_hub_points
+      - display_lidar_points
+      - linux_ros_driver
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/Livox-SDK/Livox-SDK-ROS.git
```
Comment (tfoote) [inline kinetic/distribution.yaml]: This is not a release repository generated by bloom. This whole stanza should be generated by bloom when making a release: http://wiki.ros.org/bloom/Tutorials/FirstTimeRelease please remove it for now if you just want to index your source code.

**Review by tfoote — CHANGES_REQUESTED** [review]
Please remove the release stanza as there is not yet a release repo for this.

File: kinetic/distribution.yaml (hunk 1 above)
Comment (Livox-SDK) [reply kinetic/distribution.yaml]: Thanks, tfoote!

**Comment by Livox-SDK** [comment]
Adding livox_ros_driver to documentation index for distro

**Comment by Livox-SDK** [comment]
The release stanza have been removed

**Pull request merged**